rest-server/core/src/main/scala/com/actionml/core/spark/SparkStoreSupport.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright ActionML, LLC under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * ActionML licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.actionml.core.spark
+
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.bson.codecs.configuration.CodecProvider
+
+import scala.reflect.ClassTag
+
+
+trait SparkStoreSupport {
+  def readRdd[T: ClassTag](
```

Comment:
Hmm, I assume there is some reason this has to be in a trait mixed-in to the `Store`. This should not be a public API but only used by the DAO's Spark support trait. Is there a way to protect access in some way?